src/System Application/App/Entity Text/src/EntityTextScenarioExt.EnumExt.al

```diff
@@ -0,0 +1,9 @@
+namespace System.Text;
+
+enumextension 2014 "Entity Text Scenario Ext." extends "Entity Text Scenario"
```

Comment:
I believe that this extension should life within BaseApp. For the same reasons I mentioned before, and in this particular case to follow the design on the marketing text functionallity (the extension belongs to BaseApp)

Reply:
Reminders are a base app object, thus the extension should go to the base app.